frontend/src/pages/Alerts/AlertForm/index.tsx

```diff
@@ -488,23 +488,22 @@ export const AlertForm: React.FC = () => {
 												selection={functionType}
 												setSelection={setFunctionType}
 											/>
-											{functionType !==
-												MetricAggregator.Count && (
-												<Combobox
-													selection={functionColumn}
-													setSelection={
-														setFunctionColumn
-													}
-													label="metric"
-													searchConfig={
-														searchOptionsConfig
-													}
-													onlyNumericKeys={
-														functionType !==
-														MetricAggregator.CountDistinct
-													}
-												/>
-											)}
+											<Combobox
+												selection={functionColumn}
+												setSelection={setFunctionColumn}
+												label="metric"
+												searchConfig={
+													searchOptionsConfig
+												}
+												disabled={
```

Comment:
in addition to making the combox disabled, should it have some placeholder value like `Rows` to make it more obvious that we're referring to the count of results?